masterlist.yaml

```diff
@@ -17651,6 +17656,10 @@ plugins:
       - <<: *patchProvided
         subs: [ 'Immersive Citizens - AI Overhaul' ]
         condition: 'active("Immersive Citizens - AI Overhaul.esp") and not active("CFTO - Immersive Citizens Patch.esp")'
+      - <<: *patch3rdParty
+        subs: [ '[Creation Club - Bittercup](https://www.nexusmods.com/skyrimspecialedition/mods/60587?tab=description)' ]
```

Comment:
Please shorten the url as much as possible & add a trailing slash at the end of it, so the url should look like this:
`https://www.nexusmods.com/skyrimspecialedition/mods/60587/`